client/PlayerViewModel.test.ts

```diff
@@ -151,4 +151,15 @@ describe("PlayerViewModel", () => {
 
         //expect(??).toBe(visibleCombatant1.Id);
     });
+
+    test("Player View round timer stops when encounter stops", () => {
+        encounter.AddCombatantFromStatBlock({ ...StatBlock.Default(), HP: { Value: 10, Notes: "" }, Player: "player" });
+        encounter.StartEncounter();
+        playerViewModel.LoadEncounter(encounter.GetPlayerView());
+        (<any>encounter.TurnTimer).elapsedSeconds(10);
+        (<any>playerViewModel).turnTimer.elapsedSeconds(10);
```

Comment:
In this case, it would make sense to expose TurnTimer as a public member of the PlayerViewModel since testing the knockout bindings would suck.